language-design.md

```diff
@@ -50,6 +69,9 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+The post on grayscale by Lea Verou talks about the naming of a function to produce different shades of gray. When using the gray() function, gray(0%) is black, gray(50%) is gray and gray(100%) is white. This probably is very straightforward for many people, but can be confusing sometimes. This is a clear example of why naming matters is the most important factor for Selim in the previous question. Even though gray() can be used for white, black, and (all shades of) gray, people may assume gray(100%) is another shade of gray, hence the other two functions - white(), and black() - that work the same way as gray are key in naming to avoid confusion. There, white(100%) is white, as it "gives the 'full effect' of white" [Verou, 2014], and the same with black. "The idea is that people are familiar thinking that way from grayscale printing." says Verou.
+The process of API design in this post is the way of how different functions are named and clearly documented for users and developers to read and understand, and easily use to print different shades of gray. Carson mentioned how important it was in this API that the functions were short and clear, and that if anything were to be added, it could be added later on, but is perfectly functional now.
```

Comment:
I also noted that this reading really highlighted the importance of clear documentation and the naming of functions. Even though this reading was the shortest and most informal, I personally got the most out of it. Building off your response, something that is simple as creating a name for function that produces gray can be a huge debate in API design. This shows that clear documentation and universal understanding is very important when creating an API.